localstack/services/cloudformation/cloudtrail_tracking/tests/test_cloudtrail_tracking_handler.py

```diff
@@ -0,0 +1,86 @@
+import json
```

Comment:
I'd suggest moving this to `localstack.testing.pytest`

Reply:
Nvm seems I misinterpreted the purpose of this file as this is pretty self contained as far as I can see. Not sure if we want to keep that in here at all then.